tests/unit/test_api.py

```diff
@@ -0,0 +1,51 @@
+# *****************************************************************************
```

Comment:
All of this file needs to be covered by API tests
tests needs to be in tests/api directory

Reply:
moved